server/models/Orders.ts

```diff
@@ -0,0 +1,31 @@
+import mongoose from 'mongoose'
+import uniqueValidator from 'mongoose-unique-validator'
+
+export interface Orders extends mongoose.Document {
+userID: string 
+date: Date 
+total: number 
+
+}
+
+const OrdersSchema = new mongoose.Schema<Orders>({
+  userID: {
+    type: String || mongoose.Schema.Types.ObjectId,
+    required: true,
+    unique: true,
+    ref: 'User'
+  },
+  date: {
+    type: Date,
+    required: true,
+  },
+  total: {
+    type: Number,
+    required: true,
+  }, 
+})
+
```

Comment:
This is not on the schema diagram, but we should probably have something like an orderItems field that would be an array of linked order items like this:
`
orderItems: [
{
    type: String || mongoose.Schema.Types.ObjectId,
    ref: 'OrderItem'
}
]
`
Otherwise, in the current state it seems that we won't be able to get that information from the order side